alembic/versions/2d55f804a4ed_remove_proposal_space_id_from_sync_.py

```diff
@@ -0,0 +1,35 @@
+"""remove proposal_space_id from sync_ticket
+
+Revision ID: 2d55f804a4ed
+Revises: 48ce908329c0
+Create Date: 2015-10-15 18:33:31.847676
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '2d55f804a4ed'
+down_revision = '48ce908329c0'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column, select
+from funnel.models import (SyncTicket, TicketClient)
```

Comment:
Never import from the main code in a migration. Migrations are supposed to last forever, so there could be a future in which this import will fail.